package.json

```diff
@@ -4,17 +4,37 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "test": "jest",
+    "start:server": "node server.js",
+    "test:e2e": "npm run start:server && npx cypress run",
+    "testMatch": [
+      "<rootDir>/unit-tests/**/*.test.js"
+    ],
+    "prettierCheck": "prettier --write \"src/**/*.js\"",
+    "eslintCheck": "eslint .",
+    "deploy": "gh-pages -d build",
+    "postmerge": "npm run prettierCheck && npm run deploy",
     "build": "sass src/scss:dist/css",
     "start": "sass --watch src/scss:dist/css & live-server"
   },
   "keywords": [],
   "author": "Noroff",
   "license": "ISC",
   "devDependencies": {
+    "@babel/core": "^7.20.12",
+    "@babel/plugin-transform-modules-commonjs": "^7.20.11",
+    "@babel/preset-env": "^7.20.2",
+    "babel-jest": "^29.3.1",
+    "cypress": "^12.3.0",
+    "eslint": "^8.31.0",
+    "gh-pages": "^4.0.0",
+    "jest": "^29.3.1",
     "live-server": "^1.2.2",
+    "prettier": "^2.8.3",
     "sass": "^1.54.8"
   },
   "dependencies": {
-    "bootstrap-dark-5": "^1.1.3"
+    "bootstrap-dark-5": "^1.1.3",
+    "jest-localstorage-mock": "^2.4.26"
```

Comment:
I think this is included in Jest 29 already

Reply:
Do I need to remove "bootstrap-dark-5": "^1.1.3" from line 18?